integrations/integrations.go

```diff
@@ -238,10 +238,19 @@ func startOPDevnet(
 	} else {
 		deployConfig.L1GenesisBlockTimestamp = hexutil.Uint64(time)
 	}
-	l1Allocs, err := readFromFileOrGetDefault(v.GetString(flagL1AllocsPath), opdevnet.DefaultL1Allocs)
+	// TODO thinking about removing this clunky readFromFileOrGetDefault abstraction.
```

Comment:
I think its fine to leave as is for now. If you feel like we should eventually update this lmk and I can make a new issue for it.

Reply:
Agreed. We will need to revamp the integrations package when using server/v2 I believe, so probably best to leave it be.